longevity_test.py

```diff
@@ -105,6 +108,25 @@ def test_custom_time(self):
 
         self.run_prepare_write_cmd()
 
+        # Grow cluster to target size if requested
+        if cluster_target_size := self.params.get('cluster_target_size'):
+            add_node_cnt = self.params.get('add_node_cnt')
+            node_cnt = len(self.db_cluster.nodes)
+
+            InfoEvent(message=f"Starting to grow cluster from {node_cnt} to {cluster_target_size}").publish()
+
+            while node_cnt < cluster_target_size:
+                InfoEvent(message=f"f'Adding node number {node_cnt + 1}").publish()
+                new_nodes = self.db_cluster.add_nodes(count=add_node_cnt, enable_auto_bootstrap=True)
+                self.monitors.reconfigure_scylla_monitoring()
+                up_timeout = MAX_TIME_WAIT_FOR_NEW_NODE_UP
+                with adaptive_timeout(Operations.NEW_NODE, node=self.db_cluster.nodes[0], timeout=up_timeout):
```

Comment:
we wait here for `add_node_cnt` - in case it is >1 our stats for `NEW_NODE` operations will be spoiled.
Anyway we add nodes one by one, what is the reason of adding multiple like this?

Reply:
Sorry, but I didn't understand the question

Reply:
we add multiple nodes at once `self.db_cluster.add_nodes(count=add_node_cnt, enable_auto_bootstrap=True)` what is the reason of doing like this instead of 1 node at a time?

Reply:
We don't add multiple nodes at once, since it's not supported for scylla (for now).
The only correct way is to add them one by one.

Reply:
So I'd propose to switch count to `1` in `self.db_cluster.add_nodes(count=add_node_cnt, enable_auto_bootstrap=True)`

Otherwise `with adaptive_timeout(Operations.NEW_NODE...` will collect wrong duration for `NEW_NODE` operation

Reply:
I don't understand why it would collect wrong duration, it's the same as all other add nodes we have.
Each add_node should be recorded only once.

Reply:
Well, in this context it's always 1, and when we change how scylla supports adding more than one (tablets), it will change the statstics of adding nodes anyway.